src/core/operations_problem.jl

```diff
@@ -376,31 +375,31 @@ function build_op_problem!(op_problem::OperationsProblem{M}; kwargs...) where M<
 end
 
 function _build!(psi_container::PSIContainer, template::OperationsProblemTemplate, sys::PSY.System; kwargs...)
-    verbose = get(kwargs, :verbose, true)
     transmission = template.transmission
 
     # Order is required
     #Build Services
+    # TODO: Add info print
```

Comment:
Can we also add an info print where the PSI container is created? More specifically, where the empty JuMP model is created. I'd like this because some of the solvers (e.g. Xpress#jg/new_interface) return solver log prints at that point, and in our console log it's a bit abrupt to have this empty solver log print without any context. I think this would help.

Reply:
Would it make more sense to have a @debug statement? It seems that your request is introducing a general behavior to address a specific solver output print. @daniel-thom thoughts?

Reply:
As long as a message isn't getting logged repeatedly I tend to err on the side of logging more.  If it's helpful to the user, log it.